### PR TITLE
Add File icon

### PR DESCRIPTION
**Pull request by vlaux**
#### What is the purpose of this pull request?
Add the `File` icon available on [Figma](https://www.figma.com/file/a94lX91ZmYGIDBxvb8shwLn8/00.-Styleguide---Library?node-id=10131%3A51550)

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by emersonlaurentino** [comment]
@vlux this PR must be done for the `features` branch and not for the` master`

**Comment by emersonlaurentino** [comment]
you can read more of this in this document: [CONTRIBUTING.md](https://github.com/vtex/styleguide/blob/master/.github/CONTRIBUTING.md)

**Comment by vlaux** [comment]
Thanks, gonna fix it

File: react/components/icon/File/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const iconBase = {
+  width: 16,
+  height: 16,
+}
+
+const File = ({ color, size, block }) => {
+  const newSize = calcIconSize(iconBase, size)
+
+  return (
+    <svg
```
Comment (igorbrasileiro) [inline react/components/icon/File/index.js]: Shouldn't you use the SVG component like the other icon components?

Comment (emersonlaurentino) [reply react/components/icon/File/index.js]: good point @igorbrasileiro 

really @vlaux you should use IconBase `import { Svg } from '../ IconBase'`. He is responsible for changing the icons when using admin-style.

Comment (vlaux) [reply react/components/icon/File/index.js]: Thanks for pointing it out, guys. Updated.

**Comment by emersonlaurentino** [comment]
Screen Shot 2020-04-02 at 14 25 08

@vlaux There's something weird about Icon

**Comment by estacioneto** [comment]
How about adding it already using TypeScript? 🤔

**Pull request closed without merging**